lib/deploy/storage/deploy.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+var _ = require('lodash');
+var RSVP = require('rsvp');
+
+var gcp = require('../../gcp');
+var utils = require('../../utils');
+
+module.exports = function(context, options) {
+  var files = _.get(context, 'storage.rules');
+  if (!files) {
+    return RSVP.resolve();
+  }
+
+  return gcp.storage.buckets.getDefault(options.project).then(function(defaultBucket) {
+    if (!defaultBucket) {
+      return utils.reject('Unable to locate your default bucket. Please visit the storage panel in the Firebase console to ensure provisioning.');
```

Comment:
"provisioning" is a confusing and overly technical, rephrase?

Reply:
Done.